docs/source/getting_started/getting_started.md

```diff
@@ -147,27 +164,38 @@ As a result, we will get something like:
 
 In the request, we ask the system to include the properties of the returned documents. We can use this data to implement a 'more like this' section.
 
-We also have a 'score' field which represents how well the documents match the user's interests. The higher the number, the better the documents match.
+We also have a `score` field which represents how well the documents match the user's interests. The higher the number, the better the documents match. It should be noted that the scores only have meaning in relation to other
+scores from the same requests.
+
+`snippet_id` is the id of the snippet which was found. The `document_id` in the `snippet_id` is the id of the document associated with the snippet. If you do not use ingestion options like `split` and in turn only have
```

Comment:
```suggestion
The field `snippet_id` identifies a specific snippet. The `document_id` in the `snippet_id` is the id of the document associated with the snippet. If you do not use ingestion options like `split` and in turn only have
```

Reply:
I would make `If you do not use ingestion options like split` the start of a new paragraph. This part is very important and it is easier that will be noted.